1-exercises/B-while-loop/exercise.js

```diff
@@ -6,7 +6,15 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  let i = 0;
```

Comment:
function evenNumbers(10) should log [0,2,4,6,8] in its current form.
Looking at the expected output of invoking it (line 20) for argument 3, it should log 0,2,4. 
It seems like the function evenNumbers should log the first n even numbers, where n is the argument to the function.